servlet/src/test/java/io/undertow/servlet/test/spec/UnavailableServletTestCase.java

```diff
@@ -1,41 +1,46 @@
 package io.undertow.servlet.test.spec;
 
+import io.undertow.servlet.api.ServletInfo;
 import io.undertow.servlet.test.util.DeploymentUtils;
 import io.undertow.testutils.DefaultServer;
 import io.undertow.testutils.HttpClientUtils;
 import io.undertow.testutils.TestHttpClient;
 import io.undertow.util.StatusCodes;
 import org.apache.http.HttpResponse;
 import org.apache.http.client.methods.HttpGet;
+import org.junit.Before;
+import org.junit.After;
 import org.junit.Assert;
-import org.junit.BeforeClass;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 
 import javax.servlet.ServletException;
 
 import java.io.IOException;
 
-import static io.undertow.servlet.Servlets.servlet;
-
 /**
  * @author Stuart Douglas
  */
 @RunWith(DefaultServer.class)
 public class UnavailableServletTestCase {
 
 
-    @BeforeClass
-    public static void setup() throws ServletException {
+    @Before
+    public void setup() throws ServletException {
         DeploymentUtils.setupServlet(
-                servlet("p", UnavailableServlet.class)
+                new ServletInfo("p", UnavailableServlet.class)
                         .addInitParam(UnavailableServlet.PERMANENT, "1")
                         .addMapping("/p"),
-                servlet("t", UnavailableServlet.class)
+                new ServletInfo("t", UnavailableServlet.class)
```

Comment:
what's the reason for this change?

Reply:
Reverted this change as it's unnecessary.